sdk/storage/azure-storage-blob/azure/storage/blob/_download.py

```diff
@@ -197,6 +217,7 @@ def _download_chunk(self, chunk_start, chunk_end):
             retry_active = True
             retry_total = 3
             while retry_active:
+                response: Any = None
```

Comment:
Can you just tack this `: Any` on on line 222 below instead?

Reply:
Negative
`azure\storage\blob\_download.py:222: error: only single target (not tuple) can be annotated  [syntax]
Found 1 error in 1 file (errors prevented further checking)`
MyPy blocking error 🤣